tests/unit/UCDB.py

```diff
@@ -46,9 +46,83 @@ def test_UCDB2Cobertura(self):
 		ucdbPath = Path("tests/data/ucdb.xml")
 		coberturaPath = Path("tests/data/cobertura.xml")
 
-		parser = Parser(ucdbPath)
-		model = parser.get_cobertura_model()
+		parser = Parser(ucdbPath, False)
+		model = parser.getCoberturaModel()
 
-		coberturaContent = model.get_xml()
+		coberturaContent = model.getXml()
 
 		self.assertIsNotNone(coberturaContent)
+
+		parser = Parser(ucdbPath, True)
+		model = parser.getCoberturaModel()
+
+		coberturaContent = model.getXml()
+
+		self.assertIsNotNone(coberturaContent)
+
+
+class CoverageValues(TestCase):
+	def _parseUCDB(self, ucdbPath, mergeInstances):
+		parser = Parser(ucdbPath, mergeInstances)
+		model = parser.getCoberturaModel()
+		model.getXml()
+
+		return parser, model
+
+	def test_multipleInstances(self):
+		(parser, model) = self._parseUCDB(
+			Path("tests/data/ucdb000_multiple_instances.xml"),
+			False,
+		)
+
+		self.assertEqual(parser.statementsCount, 15)
```

Comment:
Unfortunately, in Python, the expected value is the first argument.
Please switch order of parameters to `assert***` methods.

```py
self.assertEqual(14, parser.statementsCovered)
```

Reply:
Fixed in https://github.com/edaa-org/pyEDAA.UCIS/pull/7/commits/e16896577a9079c17e00b9f0b8fd368def9869cd

Reply:
Checked.